src/parser.js

```diff
@@ -722,6 +727,127 @@ export const parser = yargs(hideBin(process.argv))
       )
       .demandCommand(1, 1, "Please choose a command")
   )
+  .command("files", "Upload and manage files", (yargs) =>
+    yargs
+      .middleware(clientMiddleware)
+      .command(
+        "list",
+        "List all files",
+        groupOptions({
+          purpose: {
+            alias: "p",
+            type: "array",
+            description: "Filter listed by purpose",
+            requiresArg: true,
+            choices: FilePurposeSchema.options,
+          },
+        }),
+        async (args) => {
+          for await (const file of args.client.files()) {
+            if (!args.purpose || args.purpose.includes(file.purpose)) {
+              console.log(`${file.id} (${file.file_name})`);
```

Comment:
I am not completely satisfied with this user experience but it seems like the most compact output. Unless user repeats file names, it should be good enough. Maybe we should add `verbose` option as well?

Reply:
I think it's alright to keep it like this.